modular_skyrat/code/datums/speech_mod/_speech_mod.dm

```diff
@@ -5,13 +5,26 @@
 	for(var/datum/speech_mod/S in speech_mods)
 		if(istype(S,datumtype))
 			S.remove_speech_mod()
-			S.Destroy()
+			qdel(S)
```

Comment:
what's the change of this for?

Reply:
Nowhere in the code does anyone call `Destroy()` directly on anything. Things that are meant to be removed should be queued for deletion with `qdel(whatever)` so that the garbage subsystem takes care of it, the `Destroy()` proc is called somewhere in there afterwards.